README.md

```diff
@@ -380,11 +416,14 @@ This is an example of usage inside an ACC ViewModel:
         }
 ```
 
-The `paymentRef` will be used to determine if it's a capture from EBT Cash or EBT SNAP. If you choose to support both SNAP and EBT Cash, you will need to handle two payments in your implementation.
+The `paymentRef` is used to determine if it's a capture from SNAP or EBT Cash.
```

Comment:
This line can probably be removed or the wording should be adjusted. If we want to keep it, I would change it to something like "The `paymentRef` can be used to determine which type of tender is being captured."

Reply:
I'll adjust for now, but if it reduces complexity for developers to just remove this line, please let me know, @dleis612 ! Thank you.